doc/tag/README.md

```diff
@@ -1,4 +0,0 @@
-This folder is for generating working Doxygen tags.
```

Comment:
Per changes a few weeks ago this folder is no longer needed. All tags go to the doc/output folder now.